read-the-docs/2/conf.py

```diff
@@ -115,6 +124,10 @@ def parse_project_name_from_pom():
 html_show_sphinx = False
 html_favicon = '/doc-build/images/favicon.png'
 
+# locales
+locale_dirs = [localeDir()]
+gettext_compact = False
```

Comment:
best practice. this will make the PO files smaller, the same structure as you have for our documentation.

but you can also have one big po file per chapter. not sure what is better for you..?!

Reply:
not sure yet, good to know that we have this option. to start with I'd prefer one big :small_airplane:

Reply:
I think fat files are better. Changed it. You now have it per chapter.

![image](https://github.com/user-attachments/assets/f6893ae3-5f12-410d-b0f2-f64f3ecf2d0f)